shell/platform/windows/win32_window.cc

```diff
@@ -107,6 +107,7 @@ Win32Window::MessageHandler(HWND hwnd,
   UINT width = 0, height = 0;
   auto window =
       reinterpret_cast<Win32Window*>(GetWindowLongPtr(hwnd, GWLP_USERDATA));
+  static bool tracking_mouse_events = false;
```

Comment:
This should be an ivar, not static; someone should be able to run two completely independent Flutter window(+engine)s in the same app if they want to.

Reply:
Done